### ISSUE TITLE
simple order api client for .net core

### ISSUE THREAD

**Issue by akplm** [CyberSource/cybersource-sdk-dotnet#51]
Are you going to provide simple order api client for .net core. If yes then when? if not then, at least provide some instruction to create soa client for .net core.

**Comment by naushadqamar** [comment]
Is there any update for this request please?

**Comment by xantari** [comment]
Still no .NET Standard library. Crazy. Stripe (big competitor to Cybersource) has had .NET Core support for years now. So sad.

**Comment by xamlnative** [comment]
I was able to get the XML client API working under .NET Core. I had to strip out a lot of unnecessary stuff for the SOAP and NVP clients and add the files to a .NET Core project, but the XML client is working fine. Feel free to reach out if you want help getting this working.

**Comment by xantari** [comment]
@xamlnative Yeah thats pretty much the only way to do it (by yourself) as Cybersource is not putting much developer effort into this library unfortunately. Someone not related to cybersource did a pull request many months ago that brought this library up to .NET standard compliance, but it was never completed as Cybersource never bothered to look at it and take their changes. I looked in the pull request history and it now appears to be gone. Probably cause it was languishing out there for so long with no one bothering to make this a priority. I'll see if I can find the fork of this library to see if it is still being kept in sync.

**Comment by ilya-khorev** [comment]
> I was able to get the XML client API working under .NET Core. I had to strip out a lot of unnecessary stuff for the SOAP and NVP clients and add the files to a .NET Core project, but the XML client is working fine. Feel free to reach out if you want help getting this working.

Hi
Are you able to share this code? As we, unfortunately, need to the same

**Comment by EpiFedge91** [comment]
> I was able to get the XML client API working under .NET Core. I had to strip out a lot of unnecessary stuff for the SOAP and NVP clients and add the files to a .NET Core project, but the XML client is working fine. Feel free to reach out if you want help getting this working.

Hello - are you able to share your code?

**Comment by bill-reiss** [comment]
> > I was able to get the XML client API working under .NET Core. I had to strip out a lot of unnecessary stuff for the SOAP and NVP clients and add the files to a .NET Core project, but the XML client is working fine. Feel free to reach out if you want help getting this working.
> 
> Hello - are you able to share your code?

If I was to do this now, I would use the .NET Standard version mentioned above. If it doesn't work right out of the box, it should be pretty straightforward to get it working. Much better than all the hacks I had to do.
